### PR TITLE
Added help/account management and other updates

### PR DESCRIPTION
**Pull request by michelle-rago**
## Changes proposed in this pull request:
- Added account management
- Added external link icons
- Remove refs to "domain dashboard"
- Added info about requestor requirement

### CODE REVIEW

File: pages/help/help_account-management.md [hunk 1]
```diff
@@ -2,21 +2,66 @@
...
+You can change the contact information associated with a .gov domain you manage.
+
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li> 
+<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li>  
+<li>Click the “Manage” link for your .gov domain.</li>
+<li>Click “Your contact information” on the left-side navigation.</li>
+<li>Update your contact information.</li> 
+<li>Click “Submit.”</li>
+
```
Comment (igorkorenfeld) [inline pages/help/help_account-management.md:62]: Looks like this list missing a closing `</ol>`.  

Not a strong opinion, but I wonder if this list would be easier to maintain if we used markdown formatting in place of the HTML? That may just be down to preference though.

File: pages/help/help_domain-management.md [hunk 2]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
 
 Manage DNS information, domain contacts, and more.
  
-- [Manage DNS information for your domain](#)
-- [Add people to your domain account](#)
-- [Add a security email for your domain](#)
-- [Renew your .gov domain](#)
-- [Domain security best practices](#)
+- [Manage DNS information for your domain](#manage-DNS-information-for-your-domain)
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:15]: This anchor link isn't working. Looks like the `DNS` portion needs to be lowercase

`#manage-dns-information-for-your-domain`

File: pages/help/help_domain-management.md [hunk 3]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+- [Add or update the security email for your domain](#add-or-update-the-security-email-for-your-domain)
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:16]: This one isn't working, missing the `.gov` portion
`#add-or-update-the-security-email-for-your-.gov-domain`

File: pages/help/help_domain-management.md [hunk 4]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+- [Add people to help manage your .gov domain](#add-people-to-help-manage-your-.gov-domain)
+- [Update your contact information (email, phone)](#update-your-contact-information)
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:18]: This one isn't working either, should be
`#update-your-contact-information-(email%2C-phone)`

File: pages/help/help_domain-management.md [hunk 5]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+You can change the contact information associated with a .gov domain you manage.
+
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li> 
+<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li>  
+<li>Click the “Manage” link for your .gov domain.</li>
+<li>Click “Your contact information” on the left-side navigation.</li>
+<li>Update your contact information.</li> 
+<li>Click “Submit.”</li>
+
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:55]: Missing a closing `</ol>`

File: pages/help/help_domain-management.md [hunk 6]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+Follow these steps to add a security email for your .gov domain.
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li> 
+<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li>  
+<li>Click the “Manage” link for your .gov domain.</li>
+<li>Click “Security email” on the left-side navigation.</li>
+<li>Add or update the security email.</li> 
+<li>Click “Submit.”</li>
+
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:36]: Missing a closing `</ol>`

File: pages/help/help_domain-management.md [hunk 7]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+[Content to add](#)
+
+You will receive an email notification about your changes. 
+
+## Add or update the security email for your .gov domain
+
+Follow these steps to add a security email for your .gov domain.
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li>
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:30]: Need a closing double quote mark after `-account/` to close the href

File: pages/help/help_domain-management.md [hunk 8]
```diff
@@ -12,10 +12,46 @@ eleventyNavigation:
...
+[Content to add](#)
+
+You will receive an email notification about your changes. 
+
+## Update your contact information (email, phone)
+
+You can change the contact information associated with a .gov domain you manage.
+
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li>
```
Comment (igorkorenfeld) [inline pages/help/help_domain-management.md:49]: Need a closing double quote mark after `-account/` to close the href

File: pages/help/help_account-management.md [hunk 9]
```diff
@@ -2,21 +2,66 @@
...
+If you already have a .gov account you’ll need to have a Login.gov account to request or manage your .gov domains. Follow these steps to [create your Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}.
+
+## Update your contact information (email, phone)
+
+You can change the contact information associated with a .gov domain you manage.
+
+<ol>
+<li>If you don’t already have a Login.gov account, you’ll need to create one to manage existing .gov domains. <a href="https://login.gov/help/get-started/create-your-account/ class="usa-link usa-link--external">Create a Login.gov account</a>.</li> 
+<li>Sign in with your Login.gov account to <a href="#">manage your domain</a>.</li>
```
Comment (igorkorenfeld) [inline pages/help/help_account-management.md:57]: should it be plural, "manage your domain**s**"? If not should we update the nav and home page buttons to be singular?

**Review by igorkorenfeld — CHANGES_REQUESTED** [review]
Looking good! 

A few minor issues with links and html lists

File: pages/help/help_account-management.md [hunk 10]
```diff
@@ -2,21 +2,66 @@
...
   order: 4
   title: Account management
 ---
-
- Manage your .gov domain account.
  
-- [Get help with your Login.gov account](#)
-- [Access your .gov domain account](#)
-- [Change your contact information (email, phone)](#)
+- [Get help with your Login.gov account](#get-help-with-your-Login.gov-account)
```
[the remark this answers is not in the document]
Comment (h-m-f-t) [reply pages/help/help_account-management.md:13]: If that doesn't work, it's probably `#get-help-with-your-login-gov-account` (replacing the `.` after "login" with `-`)

File: pages/domains/domains_benefits.md [hunk 11]
```diff
@@ -54,7 +54,7 @@ If you need help coming up with your .gov domain name, [contact us](../../contac
 
 Once your .gov domain is up and running we’ll support you with these services:
 
-- **Dashboard**: Your domain dashboard will provide the status of your domain requests and information about your registered domains.
+- **Domain management**: Check the status of your domain requests and manage your registered domains.
 - **Security**: We keep you informed about [domain security best practices](../security).
 - **Domain renewal**: We make it easy to renew your .gov domain.
 - **Self-service**: You’ll be able to edit information about your contacts or your domain (like changes to your DNS settings) anytime.
```
[the remark this answers is not in the document]
Comment (michelle-rago) [reply pages/domains/domains_benefits.md:60]: I will go through the site and identify content blocks. I think even having two instances is worth it. There will def be help content that is duped between sections.

File: pages/help/help_account-management.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (michelle-rago) [reply pages/help/help_account-management.md]: Fixed

File: pages/help/help_account-management.md [hunk 12]
```diff
@@ -2,21 +2,66 @@
...
+- [Access your .gov domain account](#access-your-.gov-domain-account)
+- [Update your contact information (email, phone)](#update-your-contact-information)
```
[the remark this answers is not in the document]
Comment (michelle-rago) [reply pages/help/help_account-management.md]: Fixed

File: pages/help/help_account-management.md (hunk 1 above)
Comment (michelle-rago) [reply pages/help/help_account-management.md:60]: Yep, removing the list tags

File: pages/help/help_domain-management.md (hunk 2 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md]: Fixed

File: pages/help/help_domain-management.md (hunk 3 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md]: Fixed

File: pages/help/help_domain-management.md (hunk 4 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md]: Fixed

File: pages/help/help_domain-management.md (hunk 5 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md:54]: RM'd list tags

File: pages/help/help_domain-management.md (hunk 6 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md:36]: RM'd list tags

File: pages/help/help_domain-management.md (hunk 7 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md]: Fixed

File: pages/help/help_domain-management.md (hunk 8 above)
Comment (michelle-rago) [reply pages/help/help_domain-management.md]: Fixed

File: pages/help/help_account-management.md (hunk 9 above)
Comment (michelle-rago) [reply pages/help/help_account-management.md]: I wasn't using the phrase here to indicate the proper name of the section. I'm referring to the action. Now that you mention it though...maybe these instructions should start with "Click on 'Manage your domain'..." 🤔

**Comment by michelle-rago** [comment]
@igorkorenfeld Thanks, as always, for your close review. I'm sorry about these mistakes!

**Review by igorkorenfeld — APPROVED** [review]
Thanks for making all of the changes, looking great! And no need to apologize! 

Last thing I'll noticed is that now that those ordered lists are in markdown we can use the markdown format for linking (i.e. `[text](url)`) in place of the anchor tags. I'll leave up to you if you want to make the change here.

**Comment by michelle-rago** [comment]
> Thanks for making all of the changes, looking great! And no need to apologize!
> 
> Last thing I'll noticed is that now that those ordered lists are in markdown we can use the markdown format for linking (i.e. `[text](url)`) in place of the anchor tags. I'll leave up to you if you want to make the change here.

😄 I have a few lists on the other help page. I'll update the links and implement the content blocks at the same time.

**Pull request merged**